hugo/content/guides/building_an_extension.md

```diff
@@ -0,0 +1,47 @@
+---
+title: "Building an Extension"
+weight: 5
+---
+
+In this guide we'll be going over how to build a VSIX extension (VSCode extension) for your Langium-based language. This will allow providing LSP support in VSCode for your language. We'll assume that you've already looked at the previous guides, particularly the bundling guide, so that you're ready to build an extension. At this point we assume that your language is also working, and there are no issues running `npm run langium:generate` or `npm run build`. If there are, you'll want to correct those first.
+
+## Setting up the Scripts
+
+To get started, you'll want to have a language expressed in Langium, such as [Lox](https://github.com/langium/langium-lox) or [MiniLogo](https://github.com/langium/langium-minilogo). If you have been following along with these guides, you should already have something ready. If you don't you can also use the default language generated by the yeoman generator for Langium, presented in the [getting started](/docs/getting-started/) section.
+
+Regardless of what you're working with, you'll want to make sure you have the following scripts in your **package.json**.
+
+```json
+{
```

Comment:
I think the scripts section is nice to have.
The Linting script, is it important to add?
Here we could focus and just explain the `esbuild` command.

Reply:
Dropped the linting script.